roles/mongodb_mongod/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: Restart mongod service
```

Comment:
This should probably be variablized. A default of true would be fine I think but I think it's a good to idea to allow people to manage the restart themselves if they wish to.

Reply:
Yes, in most cases the restart would be fine but I agree that it would be a good idea to give people a choice if they wish to manage the restart themselves if they wish to. I will variablize it. Also, I will send the change for `mongodb_config` and `mongodb_mongos` as well. 
Do you think the variable name `mongodb_manual_restart` with the default value set to `false` would be good? If set to `true`, automatic service restart could be skipped.

Reply:
I think something like "skip_restart"might be more user-friendly.

Reply:
Great! I will send the changes soon :-)